FirebaseStorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
```

Comment:
There should be time to get this into next week's 10.5.0 release notes update so it can be moved into the section below. cc: @morganchen12

- [fixed] Run `pod update` or `File -> Packages -> Update to latest Packages` to update the `GTMSessionFetcher` dependency to at least version 3.1.0. This fixes an issue where it infinitely retries when FirebaseStorage returns a 500 response.

Reply:
Done.